ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -691,6 +691,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             });
           workspace = cloneWorkspace.workspace;
         } else {
+          workspace.researchPurpose.researchPurposeReviewed = true;
```

Comment:
Shouldn't this only be true if the user reviews the research purpose rather than just updating other aspects of the workspace?

Reply:
Updating other aspect will also set the researchPurpose review to true, as the pop up as a UPDATE link that will redirect the user to Workspace Edit page. 
And now i am thinking if i should put this statement inside the service rather than here which will also take care in case of LOOKS GOOD

Reply:
Right, I get that that is what the code is doing currently, but I don't think we want that - I think we only want to update this if the user actually reviews or updates the research purpose.